server/static/static.go

```diff
@@ -17,6 +18,10 @@ const (
 	indexTemplateFilename = "index.html"
 )
 
+var (
+	jsEntryPointPath = flag.String("js_entry_point_path", "/app/app_bundle.min.js", "Absolute URL path of the app JS entry point")
```

Comment:
would prefer this to be a boolean called like `minify_js` or something that defaults to true. that way we can support adding a random string / commit hash to the js path to bust the cache

Reply:
I was thinking that should be a separate flag called `--disable_cache`, since we really want to disable the cache for all resources (CSS, JSON, SVGs, etc.) and not just JS -- it would just cause the server to send `Cache-Control: no-cache` on all outgoing requests